### PR TITLE
Create start_bootstrap_admin_theme gem

### PR DESCRIPTION
**Pull request by gsmendoza**

### CODE REVIEW

**Comment by RichardJordan** [comment]
Great work George. Thanks for going to all of this effort. :+1: from me

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-test/dummy/db/*.sqlite3-*
-test/dummy/log/*.log
-test/dummy/storage/
-test/dummy/tmp/
+spec/dummy/db/*.sqlite3
+spec/dummy/db/*.sqlite3-journal
+spec/dummy/db/*.sqlite3-*
+spec/dummy/log/*.log
+spec/dummy/storage/
+spec/dummy/tmp/
```
Comment (RichardJordan) [inline .gitignore:10]: aren't you using a test directory not a spec directory? not sure I understand if this is gitignoring the right stuff

**Review by RichardJordan — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (gsmendoza) [reply .gitignore:10]: I think these paths are correct. In https://github.com/SnackNation/start_bootstrap_admin_theme/pull/1/commits/41b6b5b086081b68b885a845a39a180f100edddf, I moved the dummy app from test/ to spec/ before deleting the test directory.

**Review by gsmendoza — COMMENTED** [review]
@RichardJordan Per our discussion in Slack, I'll go merge this PR and release the gem.

**Pull request merged**